### ISSUE TITLE
use javascript in html

### ISSUE THREAD

**Issue by jellybobbin** [Keats/tera#552]
### context.insert("use_script", &"<script>alert('hello');</script>");

>How to use it to make Js take effect on the page and pop up hello

### when i use {{use_script}}  It just displays the string

**Comment by Keats** [comment]
Do `{{ use_script | safe }}`

**Comment by X-Ryl669** [comment]
Does not work for me, or I don't understand your answer.
I have this in my `.tera` file:
```
<table>
[... removed for example...]
</tbody>
</table>

<script>
    alert("Here");
</script>
```
I'm not getting any alert.
